<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>822</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButtonInitNext">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>680</y>
     <width>93</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>완료</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTrainSetTitle">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>391</width>
     <height>21</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Train</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxTest">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>699</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxValidation">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>701</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight: bold;
font-size: 18px;</string>
   </property>
   <property name="text">
    <string>Validation</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>581</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="saveDir" stretch="1,1">
    <item>
     <widget class="QLabel" name="labelModelSave">
      <property name="styleSheet">
       <string notr="true">font-weight: bold;
font-size: 18px;</string>
      </property>
      <property name="text">
       <string>Model Save Directory</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelValidationCount_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelModelSaveDir">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>90</y>
     <width>681</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>모델 저장 위치</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>570</y>
     <width>699</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="totalRatioPlot">
    <item>
     <widget class="QLabel" name="totalRatioPlotTrain_2">
      <property name="styleSheet">
       <string notr="true">font-size: 18px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Train</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="totalRatioPlotTest">
      <property name="styleSheet">
       <string notr="true">font-size: 18px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Test</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="totalRatioPlotValidation">
      <property name="styleSheet">
       <string notr="true">font-size: 18px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Validation</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>620</y>
     <width>699</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="totalRatioPlotCount">
    <item>
     <widget class="QLabel" name="totalRatioPlotCountTrain">
      <property name="styleSheet">
       <string notr="true">font-size: 18px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="totalRatioPlotCountTest">
      <property name="styleSheet">
       <string notr="true">font-size: 18px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="totalRatioPlotCountValidation">
      <property name="styleSheet">
       <string notr="true">font-size: 18px;
font-weight: bold;</string>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButtonDefTrainListDir">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>210</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>열기</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOkTrainListDir">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>170</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>열기</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOkTrainCount">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>170</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDefTrainCount">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>210</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>721</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelTotalRatioTestCount">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>290</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxTotalRatioTestCount">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>280</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelTotalRatioTestTitle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>300</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Test Set</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOkTestListDir">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>330</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>열기</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOkTestCount">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>330</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDefTestListDir">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>370</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>열기</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDefTestCount">
   <property name="geometry">
    <rect>
     <x>605</x>
     <y>370</y>
     <width>41</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>410</y>
     <width>721</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelTotalRatioValidationCount">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>450</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxTotalRatioValidationCount">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>440</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="labelTotalRatioValidationTitle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>460</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Validation Set:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOkValidationListDir">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>490</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>열기</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOkValidationCount">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>490</y>
     <width>51</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonDefValidationListDir">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>530</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>열기</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDefValidationCount">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>530</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>721</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonModelSaveDir">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>열기</string>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>560</y>
     <width>721</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="labelOkTrainListDir">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>471</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>파일 위치</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOkTrainTitle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>양품</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDefTrainTitle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>30</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>불량</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDefTrainListDir">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>210</y>
     <width>471</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>파일 위치</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOkTestTitle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>330</y>
     <width>30</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>양품</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOkTestListDir">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>471</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>파일 위치</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDefTestTitle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>30</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>불량</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDefTestListDir">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>370</y>
     <width>471</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>파일 위치</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOkValidationTitle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>490</y>
     <width>30</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>양품</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelOkValidationListDir">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>490</y>
     <width>471</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>파일 위치</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDefValidationTitle">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>530</y>
     <width>30</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>불량</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelDefValidationListDir">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>530</y>
     <width>471</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>파일 위치</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
